WNPRC_Purchasing/test/src/org/labkey/test/tests/wnprc_purchasing/WNPRC_PurchasingTest.java

```diff
@@ -319,8 +319,8 @@ public void testCreateRequestAndEmailNotification() throws IOException, CommandE
         log("Verifying the request created");
         waitForElement(Locator.tagWithAttribute("h3", "title", "Purchase Requests"));
         DataRegionTable table = DataRegionTable.DataRegion(getDriver()).find();
-        checker().verifyEquals("Invalid number of requests ", 1, table.getDataRowCount());
-        checker().verifyEquals("Invalid request status ", "Review Pending",
+        assertEquals("Invalid number of requests ", 1, table.getDataRowCount());
+        assertEquals("Invalid request status ", "Review Pending",
                 table.getDataAsText(0, "requestStatus"));
```

Comment:
I don't think it is valid to assert here, should still use checker().verifyEquals. For example the second assert could fail because of text change, that is not a reason to terminate the test and not validate the email was sent.